eq-author/src/components/PageTitle/PageTitleInput.js

```diff
@@ -54,48 +56,107 @@ const PageTitleInput = ({
   inputTitlePrefix,
   altFieldName,
   errorMessage,
-}) => (
-  <PageTitleContent>
-    <Heading data-test="page-title-input-heading">{heading}</Heading>
-    <PageDescriptionContent>
-      The page title is the first thing read by screen readers and helps users
-      of assistive technology understand what the page is about. It is shown in
-      the browser&apos;s title bar or in the page&apos;s tab. Page titles follow
-      the structure: &apos;page description - questionnaire title&apos;.
-    </PageDescriptionContent>
-    {/* From interaction designer - this will be brought back when the page titles link is fixed */}
-    {/* <p>
-      For help writing a page description, see our{" "}
-      <ExternalLink
-        url="https://ons-design-system.netlify.app/guidance/page-titles-and-urls/#page-titles"
-        linkText="design system guidance on page titles"
-      />
-    </p> */}
-    <Field data-test="page-title-missing-error">
-      <Label
-        data-test="page-title-input-field-label"
-        htmlFor={altFieldName ? altFieldName : "pageDescription"}
-      >
-        {inputTitlePrefix
-          ? `${inputTitlePrefix} page description`
-          : "Page description"}
-      </Label>
-      <StyledInput
-        id={altFieldName ? altFieldName : "pageDescription"}
-        type="text"
-        data-test="txt-page-description"
-        autoComplete="off"
-        name={altFieldName ? altFieldName : "pageDescription"}
-        placeholder=""
-        onChange={(e) => onChange(e.target)}
-        onBlur={(e) => onUpdate(e.target)}
-        value={pageDescription || ""}
-        hasError={errorMessage}
+}) => {
+  const [showPasteModal, setShowPasteModal] = useState({
```

Comment:
As this state doesn't only handle the Boolean `show` state of the paste modal, it can be renamed to describe that it handles other state values related to the paste modal, e.g. `pasteModalState`.

Reply:
I take the point, but using 'State' in the naming for useState seems redundant. I can't think of anything better.

Reply:
`pasteModalInfo` and `setPasteModalInfo` could be alternative names which don't use "state", as this state describes information related to the pasteModal.